framework/src/Volo.Abp.Ddd.Application/Volo/Abp/Application/Services/AbstractKeyReadOnlyAppService.cs

```diff
@@ -0,0 +1,185 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Dynamic.Core;
+using System.Text;
+using System.Threading.Tasks;
+
```

Comment:
You can format code and remove useless `using`,  `AbstractKeyCrudAppService` and `ReadOnlyAppService` are also.

![image](https://user-images.githubusercontent.com/16813853/85040909-e22fb700-b1bb-11ea-9b38-fbabc2b236d0.png)

Reply:
Thanks for your review. I've fixed the usings and unified the code format.